app/Http/Controllers/ReportController.php

```diff
@@ -190,6 +190,9 @@ public function destroy($id)
         $this->reportRepository->deleteFilter($id);
 
         Flash::success('Report deleted successfully.');
+        if (request()->ajax()) {
+            return $this->sendSuccess('User deleted successfully.');
```

Comment:
Why User delete in the report?
message should be like 
```suggestion
            return $this->sendSuccess('Report deleted successfully.');
```